soc/xtensa/intel_adsp/cavs_v25/Kconfig.defconfig.series

```diff
@@ -72,4 +72,8 @@ config KERNEL_VM_SIZE
 
 endif
 
+config DMA_CAVS_GPDMA
+	default y
+	depends on DMA
+
```

Comment:
This is going to be interesting how we manage the transition. In the current SOF build, this defconfig file is used, but currently Zephyr CONFIG_DMA is not set, so this gets ignored (and the DWDMA driver from SOF is used instead). But something to watch out in the future.